### PR TITLE
add doc for tekton example

### PR DESCRIPTION
**Pull request by ydayagi**
FLPATH-938
https://issues.redhat.com/browse/FLPATH-938

### CODE REVIEW

**Review by pkliczewski — COMMENTED** [review]
/lgtm

File: content/docs/workflow-examples/tekton/tekton.md
Comment (masayag) [inline content/docs/workflow-examples/tekton/tekton.md:7]: Pls don't merge until the referenced PR is merged and this link points to the parodos-dev repository.

Comment (ydayagi) [reply content/docs/workflow-examples/tekton/tekton.md:7]: other PR is merged

Comment (ydayagi) [reply content/docs/workflow-examples/tekton/tekton.md:7]: fixed the link

**Review by masayag — APPROVED** [review]

**Pull request merged**
